src/components/views/DownloadModModal.vue

```diff
@@ -152,7 +144,7 @@ import ProfileModList from '../../r2mm/mods/ProfileModList';
             tsCombo.setMod(tsMod);
             tsCombo.setVersion(tsVersion);
 
-            setTimeout(async () => {
+            await Promise.resolve().then(async () => {
```

Comment:
Nitpick: would be a bit cleaner to not do this here just to revert it in the next PR.